less/aboutUsSocialLinks.less

```diff
@@ -1,125 +1,124 @@
 @font-face {
-  font-family: 'adapt-aboutus';
-  src: url('fonts/adapt-aboutus.eot');
-  src: url('fonts/adapt-aboutus.eot') format('embedded-opentype'),
-  url('fonts/adapt-aboutus.woff') format('woff'),
-  url('fonts/adapt-aboutus.ttf') format('truetype'),
-  url('fonts/adapt-aboutus.svg') format('svg');
+  font-family: "adapt-aboutus";
+  src: url("fonts/adapt-aboutus.eot");
+  src: url("fonts/adapt-aboutus.eot") format("embedded-opentype"),
+    url("fonts/adapt-aboutus.woff") format("woff"),
+    url("fonts/adapt-aboutus.ttf") format("truetype"),
+    url("fonts/adapt-aboutus.svg") format("svg");
   font-weight: normal;
   font-style: normal;
 }
 
-
 .aboutus__sociallinks {
   display: flex;
-      padding:@item-padding;
+  padding: @item-padding;
 
   a {
     text-decoration: none;
   }
 
   .icon:before {
     color: @drawer-icon-inverted;
-          display: inline-block;
-          font-family: 'adapt-aboutus';
+    display: inline-block;
+    font-family: "adapt-aboutus";
     font-size: 2rem;
     margin: 0 0.75rem 0.75rem 0;
 
-          .no-touch &:hover {
+    .no-touch &:hover {
       color: @drawer-icon-inverted-hover;
-          }
-      }
-
-      .icon-qq:before {
-        content: "\e900";
-      }
+    }
+    color: @drawer-icon-inverted-hover;
```

Comment:
I'm only seeing that there was an original color setting on line 21 also, so better to replace that. Is the hover color what we want even on unhovered state?